openadapt/app/cards.py

```diff
@@ -37,7 +53,13 @@ async def pick_file():
     import_dialog.open()
 
 
-def recording_prompt(options, record_button):
+def recording_prompt(options: list, record_button: ui.widgets.Button) -> None:
```

Comment:
Should this be `list[...]`?

Reply:
Addressed in https://github.com/OpenAdaptAI/OpenAdapt/pull/289/commits/d88767ff785961d7eb36ca589d96f707a019b507